packages/date-picker/src/FilterDateRangePicker/components/Trigger/FilterButtonGroup/FilterButtonGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "@kaizen/component-base"
+import { FilterBaseButtonProps } from "../FilterBaseButton"
+import styles from "./FilterButtonGroup.module.scss"
+
+export interface FilterButtonGroupProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
+  children:
+    | React.ReactElement<FilterBaseButtonProps>
+    | Array<React.ReactElement<FilterBaseButtonProps>>
+}
+
+export const FilterButtonGroup: React.VFC<FilterButtonGroupProps> = ({
+  children,
+  classNameOverride,
+  ...restProps
+}) => {
+  const childCount = React.Children.count(children)
+
+  return (
+    <div
+      role="group"
+      className={classnames(styles.filterButtonGroup, classNameOverride)}
+      {...restProps}
+    >
+      {childCount === 1
+        ? children
```

Comment:
Would it be a safe assumption that if someone wanted to use `FilterButtonGroup` that it would be more than one child? Otherwise they wouldn't use it?

Reply:
Ah right, so this works if you do something like this:
```
<FilterButtonGroup>
  <Button>
  { condition && <Button> }
</FilterButtonGroup>
```

Reply:
Nope 😅 It's going to sound strange, but because you'd need Fragments for your example, it would actually fail because I didn't put in the magic recursive check to check within Fragments (I found a solution online, but didn't feel it was necessary to look into and implement yet).

```
// This has a single child, which is the fragment
<FilterButtonGroup>
  <>
    <Button>
    { condition && <Button> }
  </>
</FilterButtonGroup>
```

Which now when I think about it (after typing it out, then deleting it), it means that it doesn't work after all! Okay, no conditional buttons for anyone yet 😂 Let's put it in later when someone says they want it.

Reply:
Haha, sounds good 😄